docker/deploy.sh

```diff
@@ -29,10 +18,11 @@ docker push $REGISTRY_URI:$image_name
 echo "The image has been pushed";
 rm -rf .ssh/*
 
-if [ "$image_name" == "master" ] && [ -n "$AUTODEPLOY_URLS" ] && [ -n "$AUTODEPLOY_TOKEN" ]; then
-    IFS=';'
-    read -ra AUTODEPLOY_URL_LIST <<< "$AUTODEPLOY_URLS"
-    for url in "${AUTODEPLOY_URL_LIST[@]}"; do
-      autodeploy "$url"
-    done
+if [ "$image_name" == "master" ] && [ -n "$AUTODEPLOY_URL" ] && [ -n "$AUTODEPLOY_TOKEN" ]; then
+    # Notifying webhook
+    curl -s --output /dev/null --write-out "%{http_code}" \
+    -H "Content-Type: application/json" \
+    -X POST \
+    -d '{"token": "'$AUTODEPLOY_TOKEN'", "push_data": {"tag": "'$AUTODEPLOY_TAG'" }}' \
+    $AUTODEPLOY_URL
```

Comment:
Can we fix the indentation here or would you prefer to do that as a separate PR?

Reply:
I can fix this separately given that this is clean revert